config/locales/server.en.yml

```
@@ -244,6 +244,7 @@ en:
   topic_invite:
     failed_to_invite: "The user cannot be invited into this topic without a group membership in either one of the following groups: %{group_names}."
     user_exists: "Sorry, that user has already been invited. You may only invite a user to a topic once."
+    not_allowed: "Sorry, this user can't be invited."
```

```suggestion
    not_allowed: "Sorry, that user can't be invited."
```
(consistent with the translation above it.)